docs/tutorial.md

```diff
@@ -0,0 +1,551 @@
+# Getting Started with Spectacle: A Tutorial
+
+In this guide, we'll show you how to get started with Spectacle and walk you through the creation and customization of a presentation deck. We've created a Github repository with the [completed project](https://github.com/FormidableLabs/spectacle-boilerplate), and will link the corresponding commit where appropriate to help you follow along. If you want, you can [view the completed tutorial here](https://github.com/FormidableLabs/spectacle-boilerplate/blob/master/presentation/index.js).
+
+## 1. Set up a basic React project
+
+You can do this on your own if you'd like, or you can...
+
+- Clone down [this project](https://github.com/FormidableLabs/spectacle-boilerplate) we've started for you using `git clone git@github.com:Formidable.com/spectacle-boilerplate.git`
```

Comment:
I'd change this to instructing the user to download/remove version control like we have in the boilerplate getting started instructions, just so that people don't get confused